_includes/pbc/all/install-features/202407.0/install-the-cart-notes-feature.md

```diff
@@ -0,0 +1,146 @@
+
+
+{% info_block errorBox %}
+
+This feature integration guide expects the basic feature to be in place.
+The current feature integration guide adds the Cart Notes Backend API and Dynamic cart page update functionality.
+
+{% endinfo_block %}
+
+## Prerequisites
+
+To start feature integration, integrate the required feature:
+
+| NAME             | VERSION          | INSTALLATION GUIDE                                                                                                            |
+|------------------|------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| Order Management | {{page.version}} | [Install the Order Management feature](/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-order-management-feature.html) |
+
+## 1) Install the required modules
+
+```bash
+composer require spryker-feature/cart-notes:"{{page.version}}" --update-with-dependencies
+```
+
+Make sure the following modules have been installed:
+
+| MODULE                | EXPECTED DIRECTORY                      |
+|-----------------------|-----------------------------------------|
+| CartNotesBackendApi   | vendor/spryker/cart-notes-backend-api   |
```

Comment:
This module is not part of Cart feature - https://release.spryker.com/features/view/36
It should be installed separately.